edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/edgedeployment/EdgeDeploymentController.cs

```diff
@@ -290,13 +288,12 @@ bool IsCreatedByController(V1PersistentVolumeClaim claim)
                 return false;
             }
 
-            if (!labels.ContainsKey(KubernetesConstants.K8sEdgeDeviceLabel) || !labels.ContainsKey(KubernetesConstants.K8sEdgeHubNameLabel))
+            if (!labels.ContainsKey(KubernetesConstants.K8sEdgeDeviceLabel))
```

Comment:
What was the reason to have the hubnamelabel earlier?

Reply:
`devicename + hubname` is unique across all IoT Hubs. This would guarantee that if we selected for these labels, we would only get objects for that unique device. You could distinguish `device1 + hub1` and `device1 + hub2` at a cluster level-query by label selection. That was the original idea.

But these labels are defined by Edge on K8s and used by Edge on K8s. We didn't make any guarantees on these labels. For our own purposes, we are using these label selections as a way to limit queries to objects Edge on K8s created.